gridcapa-cse-valid-publication-app/src/main/java/com/farao_community/farao/cse_valid_publication/app/services/FileImporter.java

```diff
@@ -27,42 +24,22 @@
 public class FileImporter {
 
     private static final Logger LOGGER = LoggerFactory.getLogger(FileImporter.class);
-    private final MinioAdapter minioAdapter;
     private final UrlValidationService urlValidationService;
 
-    public FileImporter(MinioAdapter minioAdapter, UrlValidationService urlValidationService) {
-        this.minioAdapter = minioAdapter;
+    public FileImporter(UrlValidationService urlValidationService) {
         this.urlValidationService = urlValidationService;
     }
 
-    public TcDocumentType importTtcAdjustment(String ttcAdjustmentFilePath) {
-        try {
-            if (minioAdapter.fileExists(ttcAdjustmentFilePath)) {
-                InputStream inputStream = minioAdapter.getFile(ttcAdjustmentFilePath);
-                JAXBContext jaxbContext = JAXBContext.newInstance(ObjectFactory.class);
-                LOGGER.info("Importing TTC adjustment file '{}'", ttcAdjustmentFilePath);
-                return (TcDocumentType) JAXBIntrospector.getValue(jaxbContext.createUnmarshaller().unmarshal(inputStream));
-            } else {
-                LOGGER.warn("TTC adjustment file does not exist {} ", ttcAdjustmentFilePath);
-                return null;
-            }
-        } catch (Exception e) {
-            String message = String.format("Cannot retrieve TTC adjustment file %s ", ttcAdjustmentFilePath);
-            throw new CseValidPublicationInvalidDataException(message, e);
-        }
-    }
-
-    public TcDocumentType importTtcValidation(String ttcValidationUrl) {
-        try (InputStream inputStream = urlValidationService.openUrlStream(ttcValidationUrl)) {
+    public TcDocumentType importTtcFile(String ttcFileUrl, String ttcFileType) {
```

Comment:
Not sure it is very useful to mention the TTC file type at this level. I think it's good that this kind of brick only deal with I/O issues and not business related information.